pages/instances/api-cli/migrating-instances.mdx

```diff
@@ -22,6 +22,10 @@ To do so, you need the Instance’s ID and a valid API key.
 - [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
 - An [Instance](/instances/how-to/create-an-instance/), which must be stopped and not be in a placement group
 
+<Message type="important">
+  Network interface names may vary across commercial families (e.g., ENT1 vs. POP2). Ensure that any hardcoded interface names in your configurations or scripts are updated to avoid migration issues.
```

Comment:
```suggestion
  Network interface names may vary across commercial families (e.g. ENT1 vs. POP2). Ensure that any hardcoded interface names in your configurations or scripts are updated to avoid migration issues.
```